webapp/components/test-runs-query.html

```diff
@@ -205,14 +206,25 @@
           this.setProperties(batchUpdate);
         }
 
-        computeResultsRangeMessage(from, maxCount) {
-          if (from) {
-            var diff = Math.floor((Date.now() - from) / 86400000);
-            return `Showing test runs from the last ${diff} days`;
+        computeResultsRangeMessage(sha, productSpecs, from, to, maxCount) {
+          if (sha) {
+            return `Showing results from revision ${sha}`;
```

Comment:
Perhaps "results" -> "test runs" for consistency with the other messages? Or the other way around :)

Reply:
Done.